sc-memory/sc-memory/sc_keynodes.hpp

```diff
@@ -31,17 +31,17 @@ class ScKeynodes : public ScObject
   static _SC_EXTERN ScAddr const & GetRrelIndex(size_t idx);
   static _SC_EXTERN size_t GetRrelIndexNum();
 
-  SC_PROPERTY(Keynode("command_state"), ForceCreate(ScType::NodeConstClass))
-  _SC_EXTERN static ScAddr kCommandStateAddr;
+  SC_PROPERTY(Keynode("actions_state"), ForceCreate(ScType::NodeConstClass))
+  _SC_EXTERN static ScAddr kActionState;
 
-  SC_PROPERTY(Keynode("command_initiated"), ForceCreate(ScType::NodeConstClass))
-  _SC_EXTERN static ScAddr kCommandInitiatedAddr;
+  SC_PROPERTY(Keynode("action_initiated"), ForceCreate(ScType::NodeConstClass))
```

Comment:
In #357 I've asked to use modern state names, i.e. question_inititaed, etc.
action_... won't integrate with KB and all other agents right now, renaming question_ to action_ is another separate task.
For now it's better to use Action word in C++ code, but use quiestion_.. as system identifiers

Reply:
We can replace `question_` keynodes in KB with find/replace